inspirehep/records/marshmallow/authors/base.py

```diff
@@ -11,34 +11,65 @@
 from invenio_records_rest.schemas.json import RecordSchemaJSONV1
 from marshmallow import Schema, fields, post_dump
 
+from ..fields import NonHiddenRaw
 from .common import PositionSchemaV1
 
 
-class AuthorsMetadataSchemaV1(Schema):
-    """Schema for Authors records."""
+class AuthorsMetadataRawPublicSchemaV1(Schema):
+    class Meta:
+        include = {"$schema": fields.Raw()}
 
-    _collections = fields.Raw(dump_only=True)
     acquisition_source = fields.Raw(dump_only=True)
     advisors = fields.Raw(dump_only=True)
     arxiv_categories = fields.Raw(dump_only=True)
     awards = fields.Raw(dump_only=True)
     birth_date = fields.Raw(dump_only=True)
     control_number = fields.Raw(dump_only=True)
     death_date = fields.Raw(dump_only=True)
-    deleted = fields.Raw(dump_only=True)
-    email_addresses = fields.Raw(dump_only=True)
+    # deleted = fields.Raw(dump_only=True)
+    # deleted_records = fields.Raw(dump_only=True)
```

Comment:
these two fields should be added here too

Reply:
also for literature too right?

Reply:
right